src/main/java/com/gluonhq/substrate/target/AndroidTargetConfiguration.java

```diff
@@ -415,6 +423,44 @@ private String findLatestBuildTool(Path sdkPath) throws IOException {
         //  Currently, sdkmanager doesn't work with JDK 11: https://issuetracker.google.com/issues/67495440
     }
 
+    /**
+     * If android manifest and icons are present in src/android, then, this
```

Comment:
probably in a subdir?

Reply:
AndroidManifest is at src/android, icons at src/android/res